GLOSSARY.md

```diff
@@ -0,0 +1,22 @@
+# Cog Glossary
+
+## Bundle
+## Command
+### Subcommand
+## Group
+## Permission
+  * A way to define what commands can be accessed
+  * Come with bundles
+  * Can be created custom in the `site:` namespace
+
+### Examples:
+  * `operable:manage_commands`
+  * `ec2:delete`
+
+## Relay
```

Comment:
Relay - the execution environment for Cog commands (non-embedded bundled) https://github.com/operable/cog/wiki/The-Architecture-of-the-Cog-System#relay